games/frightcrawler/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        charlesrocket frightcrawler 0.3.3
+
+categories          games
+platforms           darwin
+license             MIT
+maintainers         @charlesrocket \
+                    openmaintainer
+
+description         MtG deck legality validator.
+long_description    Frightcrawler is a deck legality validator for MtG. It \
+                    reads CSV card lists from Helvault or AetherHub against provided \
+                    game format.
+homepage            https://github.com/charlesrocket/frightcrawler/
+
+depends_lib         port:crystal \
+                    port:shards
+
+checksums           rmd160  c8943ef7737817a515da9cdafba3b72d6ba2ed41 \
+                    sha256  bc5bf50045a00546bee17ad55a72fc46c381a4f808e8aa5a973c79299b686286 \
+                    size    18694
+
+use_configure       no
+
+build.cmd           ${prefix}/bin/shards install && ${build.cmd}
```

It might be useful to pass the [following flags](https://crystal-lang.org/reference/1.3/the_shards_command/index.html#shards-commands):

`--production`: Don't install developer dependencies
`--verbose`: Useful for debugging errors

```suggestion
build.cmd           ${prefix}/bin/shards --production --verbose install && ${build.cmd}
```